src/PSDocs.Azure/docs/Azure.Template.Doc.ps1

```diff
@@ -236,5 +236,16 @@ Document 'README' {
                 $example | Code 'json'
             }
         }
+        # Add command line snippet
+        if ($PSDocs.Configuration.GetBoolOrDefault('AZURE_USE_COMMAND_LINE_SNIPPET', $True)) {
+            Section $LocalizedData.CommandLine {
+                Section 'PowerShell' {
+                    'New-AzResourceGroupDeployment -Name <deployment-name> -ResourceGroupName <resource-group-name> -TemplateFile <path-to-template> -TemplateParameterFile <path-to-templateparameter>' | Code powershell
+                }
+                Section 'Azure CLI' {
```

Comment:
You could add localized string with `$LocalizedData` for PowerShell and Azure CLI section headers. Although it may not add a lot of value.

Reply:
Unless you think it's best to have separate section all together as I think both are snippets (param & CLI)